feedplanner-frontend/my-app/src/HomePage.jsx

```
@@ -0,0 +1,16 @@
+export function HomePage({isSignedOut}){
+    const email = localStorage.getItem("email")
+    return(
+        <div>
+            <header>
+                <img src="img/logo.png" alt="" />
+                <nav>
+                    <p>Welcome {email ? email : ''}</p>
```

If the email is empty, that means some error as occured right? Displaying an empty string '' might be confusing, can we throw an error if that is the case? 

Feel free to do it as a follow up if this is hard to implement right now